test/README.md

```diff
@@ -215,13 +215,6 @@ Change directory to the test directory:
 cd test
 ```
 
-To setup up minikube for drenv run once before starting any
-environment:
-
-```
-drenv setup
-```
-
```

Comment:
The issue is the missing filename argument. It is true that the setup step is not necessary at this time, since the issues it solved are already fix in current minikube version, but in the future we may have new issues requiring setup. Also we supports now lima on macOS, which may require setup in the future even if minikube does not require it.

In the future we can use drenv setup to install and upgrade the many tools that today need to be installed manually (e.g. minikube, kubectl, clusteradm, ...) or install system packages (e.g. libvirt on Linux, lima on macOS).

A better way to to fix this is to add the missing filename argument:

```console
drenv setup envs/regional-dr.yaml
```

And we can add a note to replace the environment file with another if you want to setup the system for another environment.

Reply:
Changes incorporated as suggested.